go.mod

```diff
@@ -22,59 +22,63 @@ go 1.20
 
 require (
 	github.com/TaoYang526/goutils v0.0.0-20210209083921-039008f0a57d
-	github.com/apache/yunikorn-core v0.0.0-20220325135453-73d55282f052
-	github.com/mitchellh/mapstructure v1.1.2
-	github.com/olekukonko/tablewriter v0.0.0-20170122224234-a0225b3f23b5
+	github.com/apache/yunikorn-core v0.0.0-20230609084156-d2d7ce89457c
+	github.com/apache/yunikorn-scheduler-interface v0.0.0-20230517151935-1714a5751931
+	github.com/mitchellh/mapstructure v1.5.0
+	github.com/olekukonko/tablewriter v0.0.5
 	go.uber.org/zap v1.24.0
-	gonum.org/v1/plot v0.9.0
-	gopkg.in/yaml.v2 v2.2.8
+	gonum.org/v1/plot v0.13.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Should we not move to `yaml.v3` ? The last time `yaml.v2` got updated was 3 years ago.